src/core/Track.cpp

```diff
@@ -2713,6 +2710,12 @@ void TrackView::dropEvent( QDropEvent * de )
  */
 void TrackView::mousePressEvent( QMouseEvent * me )
 {
+	if( me->x()>10 ) // 10 = The width of the grip + 2 pixels to the left and right.
```

Comment:
This breaks horizontal track resizing. I think this check should be moved to below.